app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -0,0 +1,49 @@
+module VerifySPAttributesConcern
+  def needs_verify_sp_attributes?
```

Comment:
How would you feel about renaming this `needs_sp_attribute_verification?`?

Reply:
👍